content/updates/serenity-updates-for-the-pim.json

```
@@ -0,0 +1,175 @@
+            "img": "/bundles/akeneocommunicationchannel/images/cards/new-connection-monitoring-page.png",
+            "altImg": "Connection monitoring page",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#easily-monitor-errors-on-your-connections"
+        },
+        {
+            "startDate": "June, 4th 2020",
+            "notificationDuration": 14, 
+            "tags": ["new","updates"],
+            "editions": ["CE","EE"],
+            "title": "New metrics on the Connection dashboard",
+            "description": "The Connection dashboard now displays additional information to ease error monitoring and allow you to see at a glance how your source connections are performing.",
+            "img": "/bundles/akeneocommunicationchannel/images/cards/error-monitoring-connection-dashboard.png",
+            "altImg": "Error monitoring on the Connection dashboard",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#new-metrics-on-the-connection-dashboard"
+        },
+        {
+            "startDate": "June, 4th 2020",
+            "notificationDuration": 14, 
+            "tags": ["new","updates"],
+            "editions": ["CE","EE"],
+            "title": "New association type: the two-way association",
+            "description": "Easily handle products compatibility in your PIM thanks to our **brand new two-way association type**! This new association type is now available under `Settings`/`Association types`.",
+            "img": "/bundles/akeneocommunicationchannel/images/cards/Associations_2-way-association-type.png",
+            "altImg": "Create a new two-way association type",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#new-association-type-the-two-way-association"
+        },
+        {
+            "startDate": "June, 4th 2020",
+            "notificationDuration": 14, 
+            "tags": ["new","updates"],
+            "editions": ["CE","EE"],
+            "title": "More visibility on all the jobs",
+            "description": "To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#more-visibility-on-all-the-jobs"
+        },
+        {
+            "startDate": "June, 4th 2020",
+            "notificationDuration": 14, 
+            "tags": ["new","updates"],
+            "editions": ["EE"],
+            "title": "Rules engine improvements",
+            "description": "Small UI enhancements and a new parameter for your calculation action, check out the last improvements of the rules engine!",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#rule-engine-improvements"
```

```suggestion
            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#rules-engine-improvements"
```
@christophe-chausseray A small change here too!